lib/uncruft/deprecation_handler.rb

```diff
@@ -81,6 +81,7 @@ def relative_path(absolute_path)
       Pathname.new(absolute_path)
         .relative_path_from(Rails.root).to_s
         .gsub(%r{\A(../)*vendor/cache}, '$GEM_PATH')
+        .gsub(%r{\A(vendor/bundle/ruby/\d\.\d\.\d/bin)}, '$BIN_PATH')
```

Comment:
Yeah this works for now -- I think there should be a way to find the relative path to `RbConfig::CONFIG['bindir']` and _then_ gsub the relative path out, but `vendor/cache` and `vendor/bundle` here are both relatively common conventions so it's fair to look for them explicitly too.